doc/modules/con_Assigning_approval_workflow.adoc

```diff
@@ -1,8 +1,8 @@
 // Module included in the following assemblies:
 // assembly_Administration_guide.adoc
 [id="Assigning_approval_workflow"]
-= Assigning approval workflows to portfolios
+= Setting approval processes for portfolios
 
-Assign approval workflows, once created, to your portfolios. Each approval workflow will apply to all products in a given portfolio.
+Govern access to products by setting the approval processes for the portfolios in which they are organized. Once approval processes are set for a portfolio, each product ordered in that portfolio will require approval from the groups defined in the process. Approval processes will apply to all products in a given portfolio.
```

Comment:
The last two sentences seem redundant:
1. Once approval processes are set for a portfolio, each product ordered in that portfolio will require approval from the groups defined in the process. 
2. Approval processes will apply to all products in a given portfolio.

I would keep the first one which is more descriptive.  But if you think we should keep both that is fine as well.

Otherwise the rest of the changes in the PR look good.

Reply:
@gmcculloug that is quire redundant. I've removed the last sentence.